src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -816,6 +816,11 @@ public function testGetSetMethod()
         $request->setMethod('POST');
         $request->headers->set('X-HTTP-METHOD-OVERRIDE', 'delete');
         $this->assertEquals('DELETE', $request->getMethod(), '->getMethod() returns the method from X-HTTP-Method-Override if defined and POST');
+
+        $request = new Request();
+        $request->setMethod('POST');
+        $request->query->set('_method', array('delete', 'patch'));
+        $this->assertEquals('POST', $request->getMethod(), '->getMethod() returns the request method if invalid type is defined in query');
```

Comment:
`assertSame()`

Reply:
@xabbuh 
Sorry, I just copied that from the assertion above to make no mistake. Thanks for the hint.

Should I fix this only for my new assertion or also for the others?

Reply:
changing it just here is okay IMO

Reply:
Done.